src/main/java/com/eva/logic/commands/AddApplicationCommand.java

```diff
@@ -44,6 +51,10 @@ public AddApplicationCommand(Index index, Application application) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        PanelState panelState = model.getPanelState();
+        if (!panelState.equals(APPLICANT_PROFILE) && !panelState.equals(APPLICANT_LIST)) {
```

Comment:
Does this allow the case where I am viewing Applicant A but I can add application for Applicant B?